### PR TITLE
Adding Stackrox to collection

### PR DESCRIPTION
**Pull request by jtaleric**
## Type of change

- [ ] Refactor
- [x] New feature
- [ ] Bug fix
- [ ] Optimization
- [ ] Documentation Update

## Description
Adding ACS layered product into our metric collection

## Related Tickets & Documents

- Related Issue #
- Closes #

## Checklist before requesting a review

- [ ] I have performed a self-review of my code.
- [ ] If it is a core feature, I have added thorough tests.

### CODE REVIEW

File: cmd/config/metrics-report.yml [hunk 1]
```diff
@@ -206,6 +206,24 @@
...
+
+- query: avg(avg_over_time(irate(container_cpu_usage_seconds_total{name!="", namespace="stackrox", container!="POD"}[2m])[{{.elapsed}}:])) by (container)
+  metricName: cpu-stackrox
+  instant: true
+
+- query: max(max_over_time(irate(container_cpu_usage_seconds_total{name!="", namespace="stackrox", container!="POD"}[2m])[{{.elapsed}}:])) by (container)
+  metricName: max-cpu-stackrox
+  instant: true
+
+- query: avg(avg_over_time(container_memory_rss{name!="", namespace="stackrox", container!="POD"}[{{.elapsed}}:])) by (container)
```
[the remark this answers is not in the document]
Comment (jtaleric) [reply cmd/config/metrics-report.yml:219]: interesting suggestion - can I get you to open an issue here to track that -- we might want to consider this for other components.

[the remark this answers is not in the document]
Comment (mtodor) [reply cmd/config/metrics-report.yml:219]: Done: https://github.com/kube-burner/kube-burner-ocp/issues/72

**Review by vishnuchalla — APPROVED** [review]
lgtm

**Pull request merged**